fixtures/kms/main.tf

```diff
@@ -11,8 +11,12 @@ resource "aws_kms_alias" "main" {
   target_key_id = aws_kms_key.main.key_id
 }
 
+data "aws_iam_user" "ci_s3" {
+  user_name = "TFE-S3"
+}
+
 resource "aws_kms_grant" "main" {
-  grantee_principal = var.iam_principal.arn
+  grantee_principal = data.aws_iam_user.ci_s3.arn
```

Comment:
These fixture modules, unlike the test modules, are expected to be used by anyone. So the `grantee_principal` value here should be a variable, and the data block for the `aws_iam_user` should be moved to the tests that use that user.